tests/benchmarking/test_benchmarking.py

```diff
@@ -68,6 +68,52 @@ def test_build_django_from_cache(benchmark):
         benchmark(fn)
 
 
+def test_find_descendants(large_graph, benchmark):
+    result = benchmark(large_graph.find_descendants, "mypackage")
+    assert len(result) == 28222
+
+
+def test_find_downstream_modules(large_graph, benchmark):
+    result = benchmark(large_graph.find_downstream_modules, DEEP_LAYERS[0], as_package=True)
+    assert len(result) == 80
+
+
+def test_find_upstream_modules(large_graph, benchmark):
+    result = benchmark(large_graph.find_upstream_modules, DEEP_LAYERS[0], as_package=True)
+    assert len(result) == 2159
+
+
+def test_find_shortest_chain(large_graph, benchmark):
+    result = benchmark(large_graph.find_shortest_chain, DEEP_LAYERS[0], DEEP_LAYERS[1])
+    assert result is not None and len(result) == 5
```

Comment:
I think we don't need to assert against the exact result here. This file is for benchmarks, not full tests, so a quick sanity check seems fine to me.

Reply:
Another option could be to use snapshots, so that we can assert the exact results but they are not cluttering the tests https://github.com/syrupy-project/syrupy

Reply:
It would be helpful to have it somewhere, at some point (to check feature parity between this and the Rust graph implementation) but I take your point, it could go somewhere else. Don't feel strongly about syrupy either way.

Reply:
I tried asserting against the exact result here, but it doesn't seem possible - `find_shortest_chain`/`find_shortest_chains` seem to return different results between runs? I guess there must be multiple chains of equal length?